tests/test_trainers.py

```diff
@@ -462,3 +463,9 @@ def test_without_packing(self, parallel_sizes, tmpdir):
 
     def test_with_packing(self, parallel_sizes, tmpdir):
         return self._test_sft_trainer(parallel_sizes, tmpdir, True)
+
+
+@is_trainium_test
```

Comment:
I think this is a WIP, right ?

Reply:
No it's just that pytest fails with error code 5 when it does not collect staging and trainium tests. So I just created a dummy test.